packages/notification-center/src/hooks/useFeedUnseenCount.ts

```diff
@@ -2,25 +2,20 @@ import { useQuery, UseQueryOptions } from '@tanstack/react-query';
 import type { IStoreQuery } from '@novu/client';
 
 import { useNovuContext } from './useNovuContext';
-import { FEED_UNSEEN_COUNT_QUERY_KEY } from './queryKeys';
 import type { ICountData } from '../shared/interfaces';
-import { useSetQueryKey } from './useSetQueryKey';
+import { useFeedUnseenCountQueryKey } from './useFeedUnseenCountQueryKey';
 
 export const useFeedUnseenCount = (
   { query }: { query?: IStoreQuery },
   options: UseQueryOptions<ICountData, Error, ICountData> = {}
 ) => {
   const { apiService, isSessionInitialized } = useNovuContext();
-  const setQueryKey = useSetQueryKey();
+  const feedUnseenCountQueryKey = useFeedUnseenCountQueryKey();
```

Comment:
created a few helper hooks that do return the query keys... the hooks are reused in a couple places